tutorials/foreign-object.html

```diff
@@ -0,0 +1,813 @@
+            clicking on the <code>button</code> itself will make the <code>input</code> lose focus, as <code>button</code> is one of the 
+            elements where JointJS prevents the default interaction.
+        </p>
+
+        <p>
+            If we wanted the <code>input</code> to lose focus when a user clicks on the <code>span</code> element, we can take advantage of the
+            <code>guard</code> option as follows <code>guard: (evt) => ['SPAN'].includes(evt.target.tagName)</code>. Although, including the 
+            <code>guard</code> option to manage focus in this instance would be superfluous if the <code>paper</code> event is already applied.
+        </p>
+
+        <p>
+            One of the final items you may wish to take care of when using a <code>form</code> in JointJS is to prevent the default browser behaviour 
+            when submitting user data. In order to achieve this, we will create a custom 
+            <a href="/docs/jointjs#dia.CellView.custom" target="_blank"><code>CellView</code></a> for our element. It will contain an events hash 
+            which specifies a DOM event that will be bound to a method on the view. The <i>key</i> is made up of the event name plus a CSS selector, 
+            and the <i>value</i> is a method on our custom view.
+        </p>
+
+        <p>
+            In the following example, we specify the event name <code>'submit'</code>, plus our CSS element selector <code>'form'</code>, and lastly 
+            the name of our method <code>'onSubmit'</code>. We can prevent that browser refresh by using <code>evt.preventDefault()</code>, and clear 
+            the value of the HTML input value afterwards if we wish to do so.
+        </p>
+
+        <pre><code>joint.shapes.example.FormView = joint.dia.ElementView.extend({
+
+    events: {
+        'submit form': 'onSubmit'
+    },
+
+    onSubmit: function(evt) {
+        evt.preventDefault();
+        evt.target.children.name.value = '';
+        this.model.attr('name/props/value', '');
+    },
+
+});
+</code></pre>
+
+        <h4>Props, a special attribute</h4>
+
+        <p>
+            The last point which we would like to cover when discussing HTML form controls is the special 
+            <a href="/docs/jointjs##dia.attributes.props" target="_blank"><code>props</code></a> attribute. When creating HTML elements, it's 
+            possible to define various attributes to initialize certain DOM properties. In some cases such as <code>'id'</code>, 
+            this is a one-to-one relationship, while in others such as <code>'value'</code>, the attribute doesn't reflect the property, and it
+            serves more like an initial and current state. 
+        </p>
+
+        <p>
+            When working with JointJS, it wouldn't make sense that users can only set an initial <code>'value'</code> attribute for an 
```

Comment:
Shouldn't be a separate paragraph - continues the thought from the previous paragraph.

The people reading this probably do not know what this paragraph is trying to say (that `.attr('value', 'hello')` is insufficient because it only sets the initial value, and that is why we need `.attr('props/value', 'hello')`). Try to rework it to make it clearer that `prop` special attributes allow people to set the current value of a form element, which is what we work with in JointJS.